pkg/tfbridge/schema_test.go

```diff
@@ -396,10 +396,31 @@ func TestTerraformOutputsWithSecretsSupported(t *testing.T) {
 		},
 		{
 			name: "object_property_value",
+
+			// NOTE: This input does not match the schema's type. The tfValue *should* be wrapped
+			// in a []any{}. [MakeTerraformOutputs] handles this case, but it shouldn't need to
+			// and tf should never return it.
```

Comment:
Maybe it handles it because it's trying to do MaxItems=1 mismatch toleration or something. Hard to know.